dev-libs/libbase58/libbase58-0.1.4.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A library for encoding/decoding base58"
+DESCRIPTION="libbase58 provides encoding and decoding for the base58 format, \
+used in Bitcoin addresses."
+HOMEPAGE="https://github.com/bitcoin/libbase58"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libbase58/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="97e8c3387fd4ed90208a8101e658743efbb6e21714e594f0828e6414f92c0b1d"
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libbase58$secondaryArchSuffix = $portVersion
+	lib:libbase58$secondaryArchSuffix = 0.0.2
```

Comment:
In REQUIRES, `lib:libbase58` needs to have ` compat >= 0` appended at the end of that line.
So you can either just add the " compat >= 0", or, even better, apply the following changes:
```diff
+libVersion=0.0.2
+libVersionCompat="$libVersion compat >= 0"
+
 PROVIDES="
 	libbase58$secondaryArchSuffix = $portVersion
-	lib:libbase58$secondaryArchSuffix = 0.0.2
+	lib:libbase58$secondaryArchSuffix = $libVersionCompat
 	"
```